docs/QUICKSTART.md

```diff
@@ -0,0 +1,39 @@
+# ARD-Eventhub / Quickstart
+
+This guide will help you get started with ARD-Eventhub.  
+
+No matter if you are a Publisher or Subscriber, you will need a user account to interact with the API. Request one through your contacts at SWR Audio Lab or ARD Online. Admins can reference the Users docs for account registrations.  
```

Comment:
```suggestion
No matter if you are a Publisher or Subscriber, you will need an user account to interact with the API. Request one through your contacts at SWR Audio Lab or ARD Online. Admins can reference the Users docs for account registrations.  
```

Reply:
> **"An" goes before all words that begin with vowels:**
> an apricot
> an egg
> an Indian
> an orbit
> an uprising
> **with two exceptions: When u makes the same sound as the y in you, or o makes the same sound as w in won, then a is used.**
> a union
> a united front
> a unicorn
> a used napkin
> a U.S. ship
https://amerenglish-grammar-2.webs.com/articles.htm
> a one-legged man

Reply:
nice learning - this is probably why it sounded strange to me